test/testdata/sonataflow.org_v1alpha08_sonataflow-simpleops.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     container:
       # TODO use docker.io/apache/incubator-kie-sonataflow-minimal-example:latest
       # when we complete https://github.com/apache/incubator-kie-kogito-serverless-operator/issues/504
-      image: quay.io/kiegroup/sonataflow-minimal-example:latest
+      image: kind-registry:5000/testimage/sonataflow-minimal-example:0.1
```

Comment:
We should replace this in runtime based on the result of the `SonataFlowBuild` status.

Reply:
yeap, I'll remove it all, since we are not going to push images directly to the registry